phpcs-boiler.xml

```diff
@@ -60,6 +60,8 @@
 	<!-- For help in understanding this testVersion:
 		https://github.com/PHPCompatibility/PHPCompatibility#sniffing-your-code-for-compatibility-with-specific-php-versions -->
 	<config name="testVersion" value="7.4-"/>
+	<config name="text_domain" value="madewithwpps"/>
+	<config name="prefixes" value="MadeWithWPPS"/>
```

Comment:
Part 2 of the fix.

`phpcs` expects these to be in the config, and fails without it.